jersey-servers/src/main/java/com/palantir/remoting3/servers/jersey/TraceEnrichingFilter.java

```diff
@@ -28,17 +28,27 @@
 import javax.ws.rs.container.ContainerRequestFilter;
 import javax.ws.rs.container.ContainerResponseContext;
 import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.core.Context;
 import javax.ws.rs.core.MultivaluedMap;
 import javax.ws.rs.ext.Provider;
+import org.glassfish.jersey.server.ExtendedUriInfo;
 
 @Provider
 public final class TraceEnrichingFilter implements ContainerRequestFilter, ContainerResponseFilter {
     public static final TraceEnrichingFilter INSTANCE = new TraceEnrichingFilter();
 
+    @Context
+    private ExtendedUriInfo uriInfo;
```

Comment:
how is this different from `requestContext.getUriInfo()` ?

Reply:
that returns `UriInfo` rather than `ExtendedUriInfo` which doesn't have the matched model resource